### ISSUE TITLE
Pillars of Ethernity

### ISSUE THREAD

**Issue by kalashnikovdmitryv** [ptitSeb/box86-compatibility-list#103]
### Description

Starts with extremally low FPS (~1-2 in game, ~20 in menu), unplayable

### Tested on

RPI4
Mesa 20.3.5
Box64 with Dynarec v0.1.3 2976bf5 built on Jul 12 2021 13:53:42
PI OS (Debian) upgraded to Bullseye
### Status
**Previous behavior:** 

**Current behavior:**

### Screenshots

BOX64
![2021-07-23-161059_1360x768_scrot](https://user-images.githubusercontent.com/77073943/126786677-952fb78b-b568-4796-80ca-18fd3fe172ba.png)

### OpenGL

OpenGL 3.x

**Comment by runlevel5** [comment]
I also ran into similar issue, which I reported here

https://github.com/ptitSeb/box64/issues/3537

TL;DR; the slowness is caused by thousands of small calls. Each call is quite optimized in their own rights, so we need to come up with different strategy somehow. I am unsure if the issue also curse other Unity-based games.

**Comment by ptitSeb** [comment]
> I also ran into similar issue, which I reported here
> 
> [ptitSeb/box64#3537](https://github.com/ptitSeb/box64/issues/3537)
> 
> TL;DR; the slowness is caused by thousands of small calls. Each call is quite optimized in their own rights, so we need to come up with different strategy somehow. I am unsure if the issue also curse other Unity-based games.

Not sure what kind of "small call" you are talking about here. But the point is: on regular x86 system, the game is not slow. So, there is something box64 is not doing right if the game is still slow.
